nowcasting_dataset/dataset/datasets.py

```diff
@@ -159,15 +159,15 @@ def __init__(
         self.gcs = None
         self.s3_resource = None
 
-        assert cloud in ["gcp", "aws"]
+        assert cloud in ["gcp", "aws", "local"]
```

Comment:
I wonder if we should rename `cloud` to `compute_environment` or something like that?  `cloud = 'local'` feels a bit weird.  I don't have a strong opinion though!

Reply:
If we are going for being explicit, then probably should be `compute_environment`, but our own stuff is still just our own local `cloud` too!

Reply:
Yeah, true!  So, yeah, I really don't mind what we do here.  Very happy to stick with `cloud`!